include/auto_schedule/utils.h

```diff
@@ -11,8 +11,9 @@
 
 namespace freetensor {
 
-inline std::vector<int> _randomFillArray(int total, int n,
-                                         std::default_random_engine &gen) {
+template <class RNG>
+requires std::uniform_random_bit_generator<RNG> std::vector<int>
```

Comment:
Simply
```c++
std::vector<int> _randomFillArray(int total, int n, std::uniform_random_bit_generator auto &gen) { ... }
```
should work. C++ 20 concepts can annotate `auto` typed parameters.
Below uses are the same.